atlasdb-client/src/main/java/com/palantir/atlasdb/schema/TargetedSweepSchema.java

```diff
@@ -75,6 +75,19 @@ private static Schema generateSchema() {
             conflictHandler(ConflictHandler.IGNORE_ALL);
         }});
 
+        schema.addTableDefinition("sweepProgressPerShard", new TableDefinition() {{
+            javaTableName("SweepShardProgress");
+            allSafeForLoggingByDefault();
+            rowName();
+                hashFirstRowComponent();
+                rowComponent("shard", ValueType.VAR_LONG);
+                rowComponent("sweep_conservative", ValueType.BLOB);
+            columns();
+                column("value", "v", ValueType.VAR_LONG);
+            sweepStrategy(TableMetadataPersistence.SweepStrategy.NOTHING);
```

Comment:
It seems to me like we _would_ want to sweep this table.

Reply:
Upon re-reading the design doc: or maybe cleaning these tables up will come later?